cn/lessons/basics/mix.md

```diff
@@ -0,0 +1,146 @@
+---
+
+layout: page title: Mix category: basics order: 8
```

Comment:
@cizixs I just noticed there some problems with the page meta data.  I pushed a few changes already.  I'm going to clean them up and make sure it all deploys.